README.md

```diff
@@ -584,7 +585,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Honey Patel](https://github.com/honeykpatel)
   - [Hrithik Chauhan](https://github.com/Hrithik5)
   - [Hulya Karakaya](https://github.com/hulyak)
-
+  - [Hammad Azam](https://github.com/hammadhz)
```

Comment:
```suggestion

```

why added your name here again?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
@Pradumnasaraf  asked me to add removed line & you also asked me to undo deleted line.

Reply:
Hey @hammadhz you added your profile in two places. That is why Tom is asking you to remove that remove it.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
I removed it now. you can check.